### DISCUSSION TITLE
CRF flag in ffmpeg

### DISCUSSION THREAD

**Discussion by rpdroky** [Xaymar/obs-stream-effects#877] [Ideas & Feedback]
### Discussion Thread

https://github.com/Xaymar/obs-StreamFX/wiki/Encoder-FFmpeg-NVENC

### Provide a summary of the Improvement

-qmin works but it wil brake the bitrate cap if crf is bigger then qmin is

Please consider includeing crf value support if possible

Thanks in advance , cheers !

**Comment by Xaymar** [comment]
As NVENC has no CRF rate control mode, including this option does absolutely nothing. You may want to look into VBR-CQ encoding, which is like CRF but does not achieve the same quality.

**Reply by Xaymar** [reply]
To be exact, including this option is impossible as neither FFmpeg nor NVENC has this option for NVENC.

**Reply by rpdroky** [reply]
It was a general ffmpeg tought since you mentioned h264 and h265 there , thanks allot for the reply
